maintenance.py

```diff
@@ -0,0 +1,82 @@
+import os
+import subprocess
+
+# Define your SQL Server connection details
+server = os.environ['DB_CONNECTION_URL']
+database = os.environ['DB_NAME']
+username = os.environ['DB_USERNAME']
+password = os.environ['DB_PASSWORD']
+
+#creatating procedure in DB
+def procedure_execution_on_db(query_name, query="None"):
+    try:
+        procedure_execution = subprocess.run(query_name, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
+  
+        if procedure_execution.returncode == 0:
+            print("Procedure has been created successfully")
+        else:
+         print("Error Occured:")
+         print(procedure_execution.stderr)

```

Comment:
Can you check formatting once. You use different levels of indentation. I would expect to have `print` in if and else branch on same level.

Reply:
Maybe try to use Python formatter plugin in Visual Studio code.

Reply:
it has been done vai "autopep8"